README.md

```diff
@@ -23,6 +23,7 @@ ClusterRoles are needed to allow the operators to watch multiple namespaces.
 
 ```
 oc new-project application-monitoring
+oc patch ns application-monitoring --patch '{"metadata":{"labels":{"monitoring-key":"middleware"}}}'
```

Comment:
Is there any advantage over using `oc label`: `oc label namespace application-monitoring monitoring-key=middleware`?